code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,340 @@
+/obj/item/organ/colossus
+	name = "Voice of God"
+	icon_state = "voice_of_god"
+	zone = "mouth"
+	slot = "vocal_cords"
+	actions_types = list(/datum/action/item_action/organ_action/colossus)
+
+/datum/action/item_action/organ_action/colossus
+	name = "Voice of God"
+	var/next_command = null
+	var/cooldown_stun = 900
+	var/cooldown_damage = 600
+	var/cooldown_meme = 300
+	var/cooldown_none = 150
+	var/base_multiplier = 1
+
+/datum/action/item_action/organ_action/colossus/IsAvailable()
+	if(world.time < next_command)
+		return 0
+	if(!owner)
+		return 0
+	if(!owner.can_speak())
+		return 0
+	if(check_flags & AB_CHECK_CONSCIOUS)
+		if(owner.stat)
+			return 0
+	return 1
+
+/datum/action/item_action/organ_action/colossus/Trigger()
+	. = ..()
+	var/command = stripped_input(owner, "Speak with the Voice of God", "Command", max_length = 140)
+	if(!command)
+		return
+	var/mob/living/list/listeners = list()
+	for(var/mob/living/L in get_hearers_in_view(8, owner))
+		if(!L.ear_deaf && L != owner && L.stat != DEAD)
+			listeners += L
+	if(!IsAvailable())
+		if(world.time < next_command)
+			owner << "<span class='notice'>You must wait [(next_command - world.time)/10] seconds before Speaking again.</span>"
+		return
+	var/spoken = uppertext(command)
+	owner.say(spoken, spans = list("colossus","yell"))
+	playsound(get_turf(owner), 'sound/magic/clockwork/invoke_general.ogg', 300, 1, 5)
+
+	if(!listeners.len)
+		next_command = world.time + cooldown_none
+		return
+
+	var/power_multiplier = base_multiplier
+
+	if(owner.mind)
+		//Chaplains are very good at speaking with the voice of god
+		if(owner.mind.assigned_role == "Chaplain")
+			power_multiplier *= 2
+		//Command staff has authority
+		if(owner.mind.assigned_role in command_positions)
+			power_multiplier *= 1.4
+		//Why are you speaking
+		if(owner.mind.assigned_role == "Mime")
+			power_multiplier *= 0.5
+
+	for(var/V in listeners)
+		var/mob/living/L = V
+		var/start
+		if(L.mind && L.mind.devilinfo && findtext(command, L.mind.devilinfo.truename))
+			start = findtext(command, L.mind.devilinfo.truename)
+			listeners = list(L)
+			power_multiplier *= 5 //if you're a devil and god himself addressed you, you fucked up
+			//Cut out the name so it doesn't trigger commands
+			command = copytext(command, 0, start)+copytext(command, start + length(L.mind.devilinfo.truename), length(command) + 1)
+			break
+		else if(findtext(command, L.real_name))
+			start = findtext(command, L.real_name)
+			listeners = list(L) //focus on a particular person
+			power_multiplier *= 2
+			//Cut out the name so it doesn't trigger commands
+			command = copytext(command, 0, start)+copytext(command, start + length(L.real_name), length(command) + 1)
+			break
+
+	//STUN
+	if(findtext(command, "stop") || findtext(command, "wait") || findtext(command, "stand still") || findtext(command, "hold on") || findtext(command, "halt"))
```

Comment:
I'll only accept this if it's a typecache, not a list, text processing is already ass in DM, needs a typecache to offset the cost.